src/ngMessageFormat/messageFormatCommon.js

```diff
@@ -8,15 +8,7 @@
 /* global isFunction: false */
 /* global noop: false */
 /* global toJson: false */
```

Comment:
I think this is not necessary any more (but it may have gone away in master anyway).